ansible/playbooks/roles/kafka_exporter/tasks/upgrade/main.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: Upgrade | Kafka Exporter | Check for upgrade flag file
```

Comment:
**Upgrade | Kafka Exporter |** is not needed here. See @to-bar  explanation here: https://github.com/epiphany-platform/epiphany/pull/2763#discussion_r761328303

Reply:
Done